src/Cache/PhpRedis.php

```diff
@@ -240,7 +240,7 @@ protected function expandEntry(array $values, $allow_invalid) {
     // Check expire time, allow to have a cache invalidated explicitly, don't
     // check if already invalid.
     if ($cache->valid) {
-      $cache->valid = $cache->expire == Cache::PERMANENT || $cache->expire >= REQUEST_TIME;
+      $cache->valid = $cache->expire == Cache::PERMANENT || $cache->expire >= time();
```

Comment:
Due to my fixup in generic cache unit tests in core, this usage of the REQUEST_TIME constant then caused invalid results in the get() method.